policies/templates/gcp_compute_external_ip_access_v1.yaml

```diff
@@ -71,8 +71,7 @@ spec:
             
             	# Find network access config block w/ external IP
             	instance := asset.resource.data
-            	access_config := instance.networkInterface[_].accessConfig
-            	external_ip := access_config[_].externalIp
+            	access_config := instance.networkInterfaces[_].accessConfigs
```

Comment:
We should also check that at least one access config is included. If the `accessConfigs` array is present but has no members, there isn't an external IP.

Reply:
Good call will fix today